usaspending_api/search/v2/views/spending_over_time.py

```diff
@@ -12,16 +18,30 @@
 from usaspending_api.common.api_versioning import api_transformations, API_TRANSFORM_FUNCTIONS
 from usaspending_api.common.cache_decorator import cache_response
 from usaspending_api.common.exceptions import InvalidParameterException
+from usaspending_api.common.experimental_api_flags import is_experimental_elasticsearch_api
 from usaspending_api.common.helpers.orm_helpers import FiscalMonth, FiscalQuarter, FiscalYear
-from usaspending_api.common.helpers.generic_helper import bolster_missing_time_periods, generate_fiscal_year
+from usaspending_api.common.helpers.generic_helper import (
+    bolster_missing_time_periods,
+    generate_fiscal_year,
+    min_and_max_from_date_ranges,
+    generate_fiscal_date_range,
+)
 from usaspending_api.common.validator.award_filter import AWARD_FILTER
 from usaspending_api.common.validator.pagination import PAGINATION
 from usaspending_api.common.validator.tinyshield import TinyShield
+from usaspending_api.search.v2.elasticsearch_helper import get_base_search_with_filters
 
-
-logger = logging.getLogger(__name__)
+logger = logging.getLogger("console")
 
 API_VERSION = settings.API_VERSION
+GROUPING_LOOKUP = {
```

Comment:
What prompted making this a global?

Reply:
This is also used in the integration tests to make sure that the use of those groups works as expected.

Reply:
If it's _only_ ever used outside this module by tests, and is otherwise considered only to be used by this module, I consider it okay to make it a "protected" global, and directly import that protected global into the tests (i.e. allow tests to violate that "weak private/protected" contract).

e.g. 
`_GROUPING_LOOKUP = { ... }`